docs/tests.md

```diff
@@ -27,7 +27,6 @@ Make sure that you include at least these two tests:
 
     Use the `RunTestUpgrade()` test helper to help determine compatibility with earlier module versions.
 
-    - If you must implement a breaking change to the module in a PR, include a Conventional Commit footer in one of the commits with the text `BREAKING CHANGE:` at the beginning of the footer. For more information, see the [Conventional Commits cheat sheet](https://cheatography.com/albelop/cheat-sheets/conventional-commits/).
```

Comment:
@ocofaigh @daniel-butler-irl,
 
> maybe we mention using SKIP UPGRADE TEST instead

like this? 

> Upgrade test
> 
> Use the `RunTestUpgrade()` test helper to help determine compatibility with earlier module versions.
> 
> If you need to skip the test, include `SKIP UPGRADE TEST` in the commit message and `BREAKING CHANGE: ` at the beginning of the footer of the commit message. Include information about the breaking changes.


OK?

Reply:
You don't need both. One or the other. So I suggest just `SKIP UPGRADE TEST`. However I think we need to make it clear that if skipping the upgrade test, they need to fully explain in the PR why they are doing it, and what the impact is to consumers. I don't want people skipping upgrade tests because they failed and not investigating why.

Reply:
see updated at https://github.com/terraform-ibm-modules/documentation/pull/47#discussion_r1317435050